src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -576,19 +595,27 @@ VariantContext finalizeCurrentVariant(final List<VariantContext> unmergedVariant
     }
 
     private VariantContext filterSiteByAlleleSpecificVQScore(VariantContext mergedVC,
+                                                             Map<Allele, Map<Allele, Double>> scoreMap,
                                                              Map<Allele, Map<Allele, Double>> vqScoreMap,
                                                              Map<Allele, Map<Allele, String>> yngMap,
                                                              ExtractCohort.VQScoreFilteringType vqScoreFilteringType) {
+        final Map<Allele, Double> remappedScoreMap = remapAllelesInMap(mergedVC, scoreMap, Double.NaN);
         final Map<Allele, Double> remappedVQScoreMap = remapAllelesInMap(mergedVC, vqScoreMap, Double.NaN);
         final Map<Allele, String> remappedYngMap = remapAllelesInMap(mergedVC, yngMap, VCFConstants.EMPTY_INFO_FIELD);
 
+        final Map<Allele, Double> relevantScoreMap = new LinkedHashMap<>();
+        mergedVC.getAlternateAlleles().forEach(key -> Optional.ofNullable(remappedScoreMap.get(key)).ifPresent(value -> relevantScoreMap.put(key, value)));
         final Map<Allele, Double> relevantVQScoreMap = new LinkedHashMap<>();
         mergedVC.getAlternateAlleles().forEach(key -> Optional.ofNullable(remappedVQScoreMap.get(key)).ifPresent(value -> relevantVQScoreMap.put(key, value)));
         final Map<Allele, String> relevantYngMap = new LinkedHashMap<>();
         mergedVC.getAlternateAlleles().forEach(key -> Optional.ofNullable(remappedYngMap.get(key)).ifPresent(value -> relevantYngMap.put(key, value)));
 
         final VariantContextBuilder builder = new VariantContextBuilder(mergedVC);
 
+        if (getScoreFieldName() != null) {
+            // TODO - should this be a constant in ExtractCohortLiteEngine?
```

Comment:
is this still an open question?

Reply:
No. Good catch - I *did* make it a constant!